sdcm/cluster.py

```diff
@@ -4875,6 +4875,16 @@ def get_db_nodes_cpu_mode(self):
         self.log.info("DB nodes CPU modes: %s", results)
         return results
 
+    def read_barrier(self, node: BaseNode):
+        """ Trigger read_barrier on the node
+
+        Read_barrier is functionality, which guarantees that after it finished,
+        node has applied all Raft commits, which was done in cluster before
+        read_barrier started on node.
+        """
+        with self.cql_connection_patient(node) as session:  # pylint: disable=no-member
+            session.execute("DROP TABLE IF EXISTS noexist_ks.noexist_cf;")
```

Comment:
Sorry, but hard for me to understand how the above explanation related to the code.
Please explain how dropping this table if exist related.

Reply:
Added more explanations

Reply:
sorry, but I still don't understand `read barrier` concept and can't find any docs about it. E.g. what 'read' we refer to?
I also don't understand concept of 'raft commits' - are they referred to schema or topology?

Maybe better to rename this method to 'ensure_...` or `wait_for_raft...` 

Can this cql timeout when under high load and we have some still some raft commits to apply?

Reply:
@soyacz , i updated the doc string, hope it become a bit clear. Read barrier is part of Raft feature, it will be difficult to put in doc string description for all raft feature